### PR TITLE
mimxrt/README: Added build instructions.

### PR DESCRIPTION
**Pull request by ThinkTransit**
Add basic build instructions for mimxrt

### CODE REVIEW

File: ports/mimxrt/README.md [hunk 1]
```diff
@@ -29,3 +29,36 @@ Known issues:
...
+```
+
+Arch
+``` bash
+sudo pacman -S arm-none-eabi-newlib
+```
+
+Update submodules
+``` bash
+git submodule update --init --recursive
```
Comment (jimmo) [inline ports/mimxrt/README.md:53]: This should be

```
make BOARD=... submodules
```

File: ports/mimxrt/README.md [hunk 2]
```diff
@@ -29,3 +29,36 @@ Known issues:
...
   - More peripherals (Counter, I2S, CAN, etc)
   - More Python options
+
+## Building
+
+``` bash
+cd ports/mimxrt
+```
+
+Make sure you have the appropriate c library installed
```
Comment (jimmo) [inline ports/mimxrt/README.md:39]: Maybe add the compiler package too? (It seems unlikely you'd have the compiler but not newlib?)

File: ports/mimxrt/README.md [hunk 3]
```diff
@@ -29,3 +29,36 @@ Known issues:
 TODO:
   - More peripherals (Counter, I2S, CAN, etc)
   - More Python options
+
+## Building
```
Comment (jimmo) [inline ports/mimxrt/README.md:33]: For first time building, need to also build mpy-cross.

**Review by jimmo — COMMENTED** [review]
Thanks @ThinkTransit 

Please see the CI error for the commit message (it should now include instructions on how to fix).

**Comment by ThinkTransit** [comment]
Thanks @jimmo I have updated to be more inline with other build instructions.

**Pull request closed without merging**